apps/src/pythonlab/pythonHelpers/scripts.ts

```diff
@@ -0,0 +1,17 @@
+// Find all tests in the project and run them. The test files need to start with 'test'
+// and end with '.py'. The tests be in the top-level folder or in a folder with an
+// __init__.py file (the file can be empty). An init file signifies that the folder is a module.
+// In order to restrict to find tests in a specific folder or file we can parameterize this
+// and update the start_dir and/or pattern.
```

Comment:
Do we want to go ahead and expose the ability to customize the start_dir and pattern via the JavaScript API? (I don't know how likely we are to need it.)

Reply:
We are pretty likely to need to customize `pattern` to restrict to the levelbuilder-defined validation file, so I'll parameterize the file pattern now.